test/e2e/common/pvutil.go

```diff
@@ -593,3 +594,13 @@ func createClientPod(c clientset.Interface, ns string, pvc *v1.PersistentVolumeC
 	Expect(apierrs.IsNotFound(err)).To(BeFalse())
 	return clientPod
 }
+
+func GetGCECloud() (*gcecloud.GCECloud, error) {
```

Comment:
Maybe framework/util.go would be a better place? It seems to already have a few gce methods in it.

Actually, now I'm thinking pvutil.go should go under framework/ instead of common/.  It looks like there are other _util.go files for other components there.